From fc53ee9c30b549a644e7534fa7b49ff2c87a8793 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Tue, 10 Sep 2013 14:41:44 +0200
Subject: [PATCH] Change path to WebAppMgr in pub/prv hub configuration

luna-webappmanager installs the WebAppMgr binary in /usr/sbin instead of /usr/bin as the
classic one does so we have to switch this in the hub configuration too.

Upstream-Status: Inappropiate [configuration]

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 files/conf/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/conf/CMakeLists.txt b/files/conf/CMakeLists.txt
index 640b82d..a16de13 100644
--- a/files/conf/CMakeLists.txt
+++ b/files/conf/CMakeLists.txt
@@ -36,7 +36,7 @@ set(CONF_SECURITY_DIRECTORIES_PRV "${WEBOS_INSTALL_SYSBUS_PRVROLESDIR};${WEBOS_I
 set(CONF_SECURITY_DIRECTORIES_PUB "${WEBOS_INSTALL_SYSBUS_PUBROLESDIR};${WEBOS_INSTALL_SYSBUS_DYNPUBROLESDIR}")
 set(CONF_SECURITY_MONITOR_EXE_PATH "${WEBOS_INSTALL_BINDIR}/ls-monitor")
 set(CONF_SECURITY_SYSMGR_EXE_PATH "${WEBOS_INSTALL_BINDIR}/LunaSysMgr")
-set(CONF_SECURITY_WEBAPPMGR_EXE_PATH "${WEBOS_INSTALL_BINDIR}/WebAppMgr")
+set(CONF_SECURITY_WEBAPPMGR_EXE_PATH "${WEBOS_INSTALL_SBINDIR}/WebAppMgr")
 set(CONF_SECURITY_WEBAPPMGR2_EXE_PATH "${WEBOS_INSTALL_BINDIR}/QtWebProcess")
 set(CONF_SECURITY_JS_SERVICE_EXE_PATH "js")
 set(CONF_SECURITY_MOJO_APP_EXE_PATH "mojo-app")
-- 
1.8.1.2

